schemas/oneof.md

```diff
@@ -0,0 +1,65 @@
+# Oneof
+
+As used by the [AEPs](http://aep.dev), a `oneof` represents a set of mutually
+exclusive fields.
+
+## Schema
+
+Rather than a generic schema for `oneof`, we provide common components
+supplementing the language features of specific IDLs. Using these enables
+uniform behavior (and JSON serialization) across IDLs.
+
+### protobuf
+
+Protobuf's built-in `oneof` supports mutual exclusion of fields. However, it
+does not have a way for an API indicate that at least one of the fields must be
+set. For this purpose, a protobuf `oneof` may be annotated with
+[`aep.api.OneofBehavior`](../proto/aep-api/aep/api/oneof_behavior.proto):
+
+```proto
+message Document {
```

Comment:
Please provide a corresponding JSON and JSON Schema example.

Reply:
See below.